gui/options.cpp

```diff
@@ -120,6 +122,15 @@ static const char *savePeriodLabels[] = { _s("Never"), _s("every 5 mins"), _s("e
 static const int savePeriodValues[] = { 0, 5 * 60, 10 * 60, 15 * 60, 30 * 60, -1 };
 static const char *outputRateLabels[] = { _s("<default>"), _s("8 kHz"), _s("11 kHz"), _s("22 kHz"), _s("44 kHz"), _s("48 kHz"), 0 };
 static const int outputRateValues[] = { 0, 8000, 11025, 22050, 44100, 48000, -1 };
+static const char *kbdMouseSpeedLabels[] = { _s("3"), _s("5"), _s("8"), _s("10"), _s("13"), _s("15"), _s("18"), _s("20"), 0 };
+static const int kbdMouseSpeedValues[] = { 0, 1, 2, 3, 4, 5, 6, 7, -1 };
+#ifdef PSP2
+static const char *shaderLabels[] = { _s("None"), _s("2xLCD"), _s("2xSharp"), _s("2xScan"), _s("2xAAA"), _s("2xScale"), 0 };
```

Comment:
This must be reworked. Please remove backend-specific code from the common code. Instead, provide this list via new call, just as we do for the scalers.